src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -1348,7 +1353,7 @@ class BookingService(
   fun createCas3Cancellation(
     booking: BookingEntity,
     cancelledAt: LocalDate,
-    reasonId: UUID,
+    reasonId: UUID?,
```

Comment:
Not sure why this became optional?

Reply:
Yeah, that was a hangover from another rabbit hole I went down!